engine/src/main/java/com/redhat/cloud/notifications/processors/email/EmailSubscriptionTypeProcessor.java

```diff
@@ -279,4 +284,12 @@ private void processAggregateEmailsByAggregationKey(EmailAggregationKey aggregat
             emailAggregationRepository.purgeOldAggregation(aggregationKey, endTime);
         }
     }
+
+    private boolean isSendSingleEmailForMultipleRecipientsEnabled(Set<User> users) {
+        if (ProfileManager.getLaunchMode() == NORMAL && featureFlipper.isSendSingleEmailForMultipleRecipientsEnabled()) {
+            Set<String> strUsers = users.stream().map(User::getUsername).collect(Collectors.toSet());
+            return (strUsers.contains("gduval-prod") || strUsers.contains("gduval-stage"));
```

Comment:
I wonder if we should extend our feature flipper to be able to conditionally enable features on specific accounts. Just wanting to start a discussion and I don't think it should affect this PR

Reply:
That's one of the many ideas I wish I had time to implement in the quarkus-feature-flag extension (which doesn't exist yet) 😄